develop.py

```diff
@@ -201,10 +221,28 @@ def makemigrations(migrate_plugins=True, merge=False, squash=False):
             'djangocms_flash', 'djangocms_video',
         ])
     if DJANGO_1_6:
+        from south.exceptions import NoMigrations
+        from south.migration import Migrations
+
         if merge:
             raise DjangoRuntimeWarning(u'Option not implemented for Django 1.6')
         for application in applications:
-            call_command('schemamigration', application, auto=True)
+            try:
+                Migrations(application)
+            except NoMigrations:
+                print('ATTENTION: No migrations found for {0}, creating initial migrations.'.format(application))
+                try:
+                    call_command('schemamigration', application, initial=True)
+                except SystemExit:
+                    pass
+            except ImproperlyConfigured:
+                print('WARNING: The app: {0} could not be found.'.format(application))
+            else:
+                try:
+                    with swallow_all_output():
```

Comment:
I was attempting to avoid a lot of clutter. I don't think there's any input prompts, but there will be a lot of noise and developers will miss what they need to see, which is provided by the print() lines.